packages/stk/stk_unit_tests/stk_topology/topology_test_utils.hpp

```diff
@@ -326,7 +326,8 @@ template <unsigned MAX_NODES>
 STK_INLINE_FUNCTION
 void check_side_nodes_ngp(stk::topology topology, unsigned gold_side_node_ordinals[][MAX_NODES])
 {
-  stk::mesh::Entity * allElemNodes = new stk::mesh::Entity[topology.num_nodes()];
+  constexpr unsigned MAX_NODES_PER_ELEM = 100;
```

Comment:
How do we know that the number of nodes per element is never over 100?  Is there no element in existence that would ever have over 100 nodes?

Reply:
For the element topologies defined in stk-topology, none have more than 27 nodes. But you make a good point, this is not future-proof. It's probably worth putting a check in place so that when a new developer adds some high-order element in the future, this would trip and let them know. We'll take on a task to make this more robust.